changelog/release-schedule.dd

```diff
@@ -8,24 +8,22 @@ $(UL
     $(LI Point releases are published unscheduled when important issues or regressions get fixed.)
 )
 
-$(P The release schedule for 2021 is as follows:)
+$(P The release schedule for 2022 is as follows:)
 
     $(DIVC release-schedule,
         $(TABLE
-            $(BETA_RELEASE 2020-12-15, 2.095.0)
-            $(MINOR_RELEASE 2021-01-01, 2.095.0)
-            $(BETA_RELEASE 2021-02-15, 2.096.0)
-            $(MINOR_RELEASE 2021-03-01, 2.096.0)
-            $(BETA_RELEASE 2021-04-15, 2.097.0)
-            $(MINOR_RELEASE 2021-05-01, 2.097.0)
-            $(BETA_RELEASE 2021-06-15, 2.098.0)
-            $(MINOR_RELEASE 2021-07-01, 2.098.0)
-            $(BETA_RELEASE 2021-08-15, 2.099.0)
-            $(MINOR_RELEASE 2021-09-01, 2.099.0)
-            $(BETA_RELEASE 2021-10-15, 2.100.0)
-            $(MINOR_RELEASE 2021-11-01, 2.100.0)
-            $(BETA_RELEASE 2021-12-15, 2.101.0)
-            $(MINOR_RELEASE 2022-01-01, 2.101.0)
+            $(BETA_RELEASE 2022-02-15, 2.099.0)
+            $(BETA_RELEASE 2022-03-01, 2.099.0)
+            $(BETA_RELEASE 2022-04-15, 2.100.0)
+            $(MINOR_RELEASE 2022-05-01, 2.100.0)
+            $(BETA_RELEASE 2022-06-15, 2.101.0)
```

Comment:
I'll be moving the release process into a more automated set-up, so it'll likely be `2022-07-15` that the next beta will be made.

Reply:
And will releases then continue in even numbered months?

Reply:
Yep.

Reply:
I changed it